apps/studio/components/layouts/ProjectLayout/LayoutHeader/HelpPopover.tsx

```diff
@@ -44,42 +48,70 @@ export const HelpPopover = () => {
-            For issues with your project hosted on supabase.com, or other inquiries about our hosted
-            services.
+            For issues with your project hosted on supabase.com or other hosted service inquiries.
+            Response times are based on your billing plan, with paid plans prioritized.
           </p>
-          <div className="space-x-1">
-            <Button asChild type="default" icon={<Wrench />}>
-              <Link
-                href="https://supabase.com/docs/guides/platform/troubleshooting"
-                target="_blank"
-                rel="noreferrer"
+        </div>
+        <div className="px-5">
+          <ButtonGroup className="w-full">
+            {projectRef && (
+              <ButtonGroupItem
+                size="tiny"
+                icon={<AiIconAnimation allowHoverEffect size={14} />}
+                onClick={() => {
+                  snap.newChat({
+                    name: 'Support',
+                    open: true,
+                    initialInput: `I need help with my project`,
+                    suggestions: {
+                      title:
+                        'I can help you with your project, here are some example prompts to get you started:',
+                      prompts: [
+                        'Summarise my database health and performance',
+                        'View and debug my edge function logs',
+                        'Implement row level security for my tables',
+                      ],
+                    },
+                  })
+                }}
               >
-                Troubleshooting
-              </Link>
-            </Button>
-            <Button asChild type="text" size="tiny" icon={<BookOpen />}>
-              <Link href="https://supabase.com/docs/" target="_blank" rel="noreferrer">
-                Docs
-              </Link>
-            </Button>
-            <Button asChild type="text" size="tiny" icon={<Activity />}>
-              <Link href="https://status.supabase.com/" target="_blank" rel="noreferrer">
-                Supabase Status
-              </Link>
-            </Button>
-          </div>
-          <p className="text-sm text-foreground-lighter">
-            Expected response time is based on your billing plan. Projects on paid plans are
-            prioritized.
-          </p>
-          <div>
-            <Button asChild type="default" icon={<Mail />}>
-              <Link href={supportUrl}>Contact Support</Link>
-            </Button>
-          </div>
+                Supabase Assistant
+              </ButtonGroupItem>
+            )}
+            <ButtonGroupItem
+              size="tiny"
+              icon={<Wrench strokeWidth={1.5} size={14} />}
+              onClick={() =>
+                window.open('https://supabase.com/docs/guides/platform/troubleshooting', '_blank')
+              }
+            >
+              Troubleshooting
+            </ButtonGroupItem>
```

Comment:
nit: could we use `<a>` tags here instead of manually calling window.open? from an accessibility POV

Reply:
so perhaps ButtonGroupItem needs to support an `asChild` param